airflow-core/src/airflow/ui/src/constants/reprocessBehaviourParams.ts

```diff
@@ -16,8 +16,23 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+import type { ReprocessBehavior } from "openapi/requests/types.gen";
+
 export const reprocessBehaviors = [
   { label: "backfill.missingRuns", value: "none" },
   { label: "backfill.missingAndErroredRuns", value: "failed" },
   { label: "backfill.allRuns", value: "completed" },
 ];
+
+export const mapReprocessBehavior = (param: string): ReprocessBehavior => {
+  switch (param) {
+    case "all_runs":
+      return "completed";
+    case "missing_and_errored_runs":
+      return "failed";
+    case "missing_runs":
+      return "none";
+    default:
+      return "none";
```

Comment:
Default can be used for the missing as well
```suggestion
    default:
      return "none";
```